sound/soc/sof/imx/Kconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
```

Comment:
Is this intentional or a copy/paste from Intel stuff?

Reply:
@plbossart copy/paste. Actually I'm not good with licenses at all. Let me consult internally with legal and might come back with the same license Intel used.